tests/5.1/scope/test_scope_private_construct.c

```diff
@@ -0,0 +1,38 @@
+//--------------- test_scope_private_construct.c -----------------------------//
+//
+// OpenMP API Version 5.1 Nov 2020
+//
+// This test checks that the scope private construct clause is properly working.
+// The test itself passes a test integer into the scope pragma and ensures that
+// all changes made to it are not kept outside of the scope region.
+//----------------------------------------------------------------------------//
+
+
+#include <omp.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "ompvv.h"
+#include <math.h>
+
+int test_scope(){
+	int errors;
+	int test_int = 1;
+	#pragma omp parallel firstprivate(test_int)
+	{
+		#pragma omp scope private(test_int)
+		{
+			test_int += 1;
```

Comment:
GCC rightly complains:
```
test_scope_private_construct.c:24:34: warning: ‘test_int’ is used uninitialized [-Wuninitialized]
   24 |                         test_int += 1;
      |                         ~~~~~~~~~^~~~
```
as `i` is private and not `firstprivate`. Should this be an assignment instead?